### DISCUSSION TITLE
May I ask if it supports Chinese

### DISCUSSION THREAD

**Discussion by wdm0814** [elsa-workflows/elsa-core#4474] [Q&A]
May I ask if it supports Chinese

**Comment by sfmskywalker** [comment]
Hi @wdm0814 !

Internationalization / localization isn't implemented yet, but we have an issue for it [here](https://github.com/elsa-workflows/elsa-studio/issues/23).

**Comment by sfmskywalker** [comment]
Hi @wdm0814 

[Elsa Studio](https://github.com/elsa-workflows/elsa-studio) now supports localization, and we're looking for contributors to help create translations. We use [Weblate](https://hosted.weblate.org/projects/elsa-studio/) to make managing translations simple and collaborative. If you're interested in contributing Chinese translations, please sign up for a Weblate account and share your username or email address with me.

We'd greatly appreciate your help!
